logging/src/commonMain/kotlin/DispatchLogger.kt

```diff
@@ -3,45 +3,74 @@ package com.juul.tuulbox.logging
 import kotlinx.atomicfu.atomic
 import kotlinx.atomicfu.update
 
+private data class DispatcherState(
+    val consumers: Set<Logger>,
+) {
+    val logLevel: LogLevel by lazy {
+        consumers.maxOfOrNull { it.minimumLogLevel }
+            ?: LogLevel.Assert
+    }
+
+    companion object {
+        val EMPTY = DispatcherState(emptySet())
+    }
```

Comment:
nit

JetBrains [discourages](https://discuss.kotlinlang.org/t/best-practices-for-top-level-declarations/2198/3) using `object`s purely for namespacing purposes.
Could just have as a top-level `private val`?